packages/webpack/src/config/base.js

```diff
@@ -18,6 +19,8 @@ import WarningIgnorePlugin from '../plugins/warning-ignore'
 
 import { reservedVueTags } from '../utils/reserved-tags'
 
+const BUILD_HASH = crypto.createHash('md5').update(new Date().toISOString()).digest('hex').substr(0, 20)
```

Comment:
Different builds will be same hash in dev reloading.

Reply:
That's right, haven't thought about dev at all here. Do you think it'd be clever to refresh the build hash on dev?

Reply:
If user have any code differing from builds, it should work for both dev and production, or they can’t verify the functionality in dev